package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "assets",
-  "version": "1.0.0",
-  "description": "The package impements clientside asset (such as CSS and JavaScript) management for PHP.",
-  "author": "yiisoft",
-  "license": "BSD-3-Clause",
-  "bugs": {
-    "url": "https://github.com/yiisoft/assets/issues"
-  },
-  "homepage": "https://github.com/yiisoft/assets#readme",
-  "devDependencies": {
-    "sass": "^1.26.9"
-  }
+    "name": "assets",
```

Comment:
Why do we need `package.json` at all?

Reply:
To install sass and test the converter with a cli, and increase the code coverage, I can remove it but it will lower the code coverage.